eq-author-api/schema/typeDefs.js

```diff
@@ -70,6 +70,7 @@ type History {
   publishStatus: String!
   questionnaireTitle: String!
   bodyText: String
+  type: String
```

Comment:
Should this be required instead of optional?

Reply:
Should also be an enum of its available options.